include/fiction/utils/layout_utils.hpp

```diff
@@ -350,7 +352,54 @@ Lyt convert_to_fiction_coordinates(const sidb_cell_clk_lyt_siqad& lyt) noexcept
 
     return lyt_new;
 }
+/**
+ * Generates a random coordinate within the region spanned by two given coordinates. The two given coordinates form the
+ * lower left corner and the upper right corner of the spanned region.
+ *
+ * @OffsetCoordinateType The coordinate implementation to be used.
+ * @param coordinate1 Coordinate of the lower left corner.
+ * @param coordinate2 Coordinate of the upper right corner (coordinate order is not important, automatically swapped if
```

Comment:
In fiction, coordinates usually increase in value from top left to bottom right. Is there a reason that coordinates in this function are seemingly handled differently?